src/containers/ServiceReview.js

```diff
@@ -0,0 +1,40 @@
+/** @jsx jsx */
+import { jsx } from '@emotion/core';
+import { Fragment } from 'react';
+import { useHistory } from 'react-router-dom';
+
+import { Routes } from 'constants/Routes';
+import { routeWithParams } from 'lib/utils/routes';
+import RequestKinds from 'lib/organizations/kinds';
+
+import GroceryServiceReview from 'components/ServiceReview/Grocery';
+
+function ServiceReview({ id, service }) {
+  const history = useHistory();
+
+  const handleSubmit = () => {
+    history.push(
+      routeWithParams(Routes.SERVICE_CONFIRMATION, {
+        id,
+      }),
+    );
+  };
+
+  if (!service) {
+    return null;
+  }
```

Comment:
One thing you can do here is use the `PageLoader` pattern we have in some pages. If you take this `return null` logic out and then in `service_review.js` you put something like:
```
const [isLoading, setIsLoading] = useState(true);
useEffect(() => {
  backend.getServiceRequest(params.id).then((response) => {
    setService(response);
    setIsLoading(false);
  });
}, [params.id, backend]);

return (
  <Page currentProgress={5} totalProgress={5}>
    {isLoading && <PageLoader />}
    {!isLoading && (
      <ServiceReview backend={backend} id={params.id} service={service} />
    )}
  </Page>
);
```

This way, empty values never make it into the component. 

You can see the PageLoader pattern on these pages:
https://github.com/helpsupply/helpsupply/blob/master/src/pages/contact.js
https://github.com/helpsupply/helpsupply/blob/master/src/pages/dashboard.js

Reply:
just a heads up i removed this on the delete request because when this happens it sends the user to the top of the page which isn't ideal if they have requests that scroll